src/hid/hid-types.h

```diff
@@ -70,7 +70,7 @@ namespace librealsense
     };
 #pragma pack(pop)
 
-    static_assert(sizeof(REALSENSE_HID_REPORT) == SIZE_OF_HID_IMU_FRAME, "HID IMU Frame struct expected size is 32 bytes");
+    //static_assert(sizeof(REALSENSE_HID_REPORT) == SIZE_OF_HID_IMU_FRAME, "HID IMU Frame struct expected size is 32 bytes");
```

Comment:
Remove comment and const SIZE_OF_HID_IMU_FRAME

Reply:
Done